man/docker-network-connect.1.md

```diff
@@ -0,0 +1,38 @@
+% DOCKER(1) Docker User Manuals
+% Docker Community
+% OCT 2015
+# NAME
+docker-network-connect - connect a container to a network
+
+# SYNOPSIS
+**docker network connect NAME CONTAINER**
+
+[**--help**]
+
+# DESCRIPTION
+
+Connects a running container to a network. Once connected, the container can
```

Comment:
does the container have to be running, or can any existing container be connected to a network?